std_msgs/String action_type
Propkey[] propkeys
